lighthouse-core/test/chromium-web-tests/web-test-server.sh

```diff
@@ -17,7 +17,7 @@ export latest_content_shell="$LH_ROOT/.tmp/chromium-web-tests/content-shells/$la
 #   - /inspector-sources -> the inspector resources from the content shell
 #   - CORS (Access-Control-Allow-Origin header)
 
-ln -s "$DEVTOOLS_PATH/out/Default/resources/inspector" "$DEVTOOLS_PATH/test/webtests/http/tests/inspector-sources"
+ln -s "$latest_content_shell/out/Release/resources/inspector" "$DEVTOOLS_PATH/test/webtests/http/tests/inspector-sources"
```

Comment:
How I debugged...

I removed the output redirection of the webserver to dev/null and noticed 404s. integration_runner.js was missing. I realized line wasn't doing what the comment suggested... the `inspector-sources` file is meant to come from content_shell, the reason is that the compiled devtools framework code should match the content shell binary.

I guess `integration_runner.js` was being generated by devtools before, but no longer is. the incorrect symbolic link was overlooked because of that.

Reply:
Scratch that. DT team refactored the BUILD rules, and so we weren't building the right thing any more. And the output folder changed slightly.